accounts/checkers.go

```diff
@@ -549,6 +552,13 @@ func checkSend(ctx context.Context, chainParams *chaincfg.Params,
 		return fmt.Errorf("error validating account balance: %w", err)
 	}
 
+	if len(paymentHash) != 0 {
```

Comment:
`paymentHash` is an array, so its length is always set! Need to compare it to an all-zero hash instead: `if paymentHash != lntypes.Hash{} {`